src/main/resources/language.yml

```diff
@@ -101,7 +101,7 @@ flower:
   FLOWERING_AZALEA: "§2Flowering Azalea"
   AZALEA_LEAVES: "§2Azalea Leaves"
   FLOWERING_AZALEA_LEAVES: "§2Flowering Azalea Leaves"
-  GLOW_BERRIES: "§2Glow Berries"
+  CAVE_VINES: "§2Cave Vines"
```

Comment:
Is cave vines here really a better name? The icon is glow berries, and the vines are generally referred to as glow berries. Cave Vines is only the actual placed block. 

The community generally knows the block as glow berries, so we should keep that name